src/oauth2.ts

```diff
@@ -94,12 +110,27 @@ export default class OAuth2 {
     } else {
       this._transport = new Transport();
     }
+    if (useVerifier === true) {
```

Comment:
any reason why you need to explicitly checks that it equals `true`? as opposed to just `if (useVerifier)`